src/components/freshness-bar/FreshnessBar.tsx

```diff
@@ -0,0 +1,46 @@
+import { differenceInMilliseconds, format } from 'date-fns';
+import React from 'react';
+import * as R from 'ramda';
+
+import Tooltip from '../tooltip';
+import { Bar } from './Bar';
+
+type BarSizes = 'sm' | 'md';
+
+export interface Props {
+  /** expiration date given as `Date`(e.g. via new Date(model.createdAt)) */
+  expirationDate: Date;
+  /** Size, short(sm) or medium (md) */
+  size?: BarSizes;
```

Comment:
old name was `length` and received `m` and `s`, we have no pattern like that in the project
now it's `size` and uses `md`, `sm` as we have in other components